front/src/applications/operationalStudies/components/Study/AddOrEditStudyModal.tsx

```diff
@@ -189,41 +189,63 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
               <div className="col-xl-6">
                 <div className="study-edition-modal-type mb-2">
                   <SelectImprovedSNCF
-                    title={
+                    label={
                       <div className="d-flex align-items-center">
                         <span className="mr-2">
                           <RiQuestionLine />
                         </span>
                         {t('studyType')}
                       </div>
                     }
-                    selectedValue={{
-                      key: currentStudy?.study_type,
-                      value: t(`studyCategories.${currentStudy?.study_type || 'nothingSelected'}`),
-                    }}
+                    value={
+                      currentStudy
+                        ? {
+                            id: currentStudy.study_type,
+                            label: t(
+                              `studyCategories.${currentStudy.study_type || 'nothingSelected'}`
+                            ).toString(),
+                          }
+                        : undefined
+                    }
                     options={studyCategoriesOptions}
-                    onChange={(e) => setCurrentStudy({ ...currentStudy, study_type: e.key })}
+                    onChange={(e) =>
+                      setCurrentStudy({
+                        ...currentStudy,
+                        study_type: e?.id as StudyUpsertRequest['study_type'],
+                      })
+                    }
                     data-testid="studyType"
                   />
                 </div>
               </div>
               <div className="col-xl-6">
                 <div className="study-edition-modal-state mb-2">
                   <SelectImprovedSNCF
-                    title={
+                    label={
                       <div className="d-flex align-items-center">
                         <span className="mr-2">
                           <FaTasks />
                         </span>
                         {t('studyState')}
                       </div>
                     }
-                    selectedValue={{
-                      key: currentStudy?.state,
-                      value: t(`studyStates.${currentStudy?.state || 'nothingSelected'}`),
-                    }}
+                    value={
+                      currentStudy
+                        ? {
+                            id: currentStudy.state,
+                            label: t(
+                              `studyStates.${currentStudy.state || 'nothingSelected'}`
+                            ).toString(),
+                          }
+                        : undefined
+                    }
```

Comment:
You've right ...
BTW, it's weird for me to have a default empty value instead of a placeholder. 
But the component doesn't accept placeholders. It can be an improvement for later